backend/models/membership.js

```diff
@@ -14,13 +14,17 @@ const membershipSchema = new mongoose.Schema({
         index: true,
         isRequired: true
     },
+    firstName: {
+        type: String,
+        required: true
+    },
     role: String,
     createdAt: {
         type: Date,
         default: Date.now
     }
 });
 
-const Membership = mongoose.model('Membership', membershipSchema);
+const Membership = mongoose.model('Membership', membershipSchema, 'users');
```

Comment:
what is the `users` for?

Reply:
I am selecting the user's collection. As for the auth state, maybe? I didn't see anything about persisting the user login without sending a token so the web browser could identify itself when it navigates away. I don't understand how to do it like that without a token or authorization other than by logging in on every page.

Reply:
The reason I asked about that is because the membership mongoose data model is for the `organization` to `user` relationship. If a user is an organization they would have a membership listed in that table. but if they user is not then there would not be any membership listed for that user.

Reply:
I was having issues without it: the query to Mongo DB wouldn't find users that existed, and that allowed it when I passed the users. I can make another model to user there, maybe?

Reply:
But my question is why access the `Membership` table?

Reply:
Finally, is there a work around we could use for this? How can I help with getting the data you would need instead of storing the membership model into `users` collection.

Reply:
const users = await User.findOne({ accountId: user.uid });
        if (!users) {
            return res.status(404).send({
                status: 'error',
                message: 'User not found',
            });
        }
That's left over, I believe. The users are coming from this now, so the users are not doing anything in your code snippet anything now and can just be deleted from that model.

 const membership = await Membership.findOne({ accountId: user.uid }); nothing is being passed anymore when it's called either. I think I just forgot to remove it there as well.

Reply:
@Donnyd88, ok sounds good, I will make that final commit, push, and then approve this PR 👍🏽

Reply:
awesome. I'm just going to wrap the org and users page in the protected route and merge it if you haven't done that already once you've done that.

Reply:
ok, sounds good 👍🏽,  considering that I renamed the files too, it makes it easier to use `useContext` now.